service/src/main/resources/example.application.yaml

```diff
@@ -6,6 +6,7 @@ logingw:
       alg-name: "RS256"
     config:
       some-key: "BETA"
+      git-properties: false
```

Comment:
This key would deserve a comment on how it works (this is to generate on startup, but will not be loaded unless restarted)

Reply:
Did So now